terraform/paas/staging.env.tfvars

```diff
@@ -1,10 +1,23 @@
-paas_space                = "get-into-teaching"                # Currently Set to Development while waiting on Staging Service Now ticket to be actioned
+paas_space                = "get-into-teaching-test"            # Currently Set to Development while waiting on Staging Service Now ticket to be actioned
 paas_database_common_name = "school-experience-staging-pg-common-svc"
 paas_redis_1_name         = "school-experience-staging-redis-svc"
 paas_application_name     = "school-experience-app-staging"
-paas_internet_hostnames   = ["staging-school-experience"]
+paas_internet_hostnames   = []
 application_instances     = 1
 environment               = "staging"
 azure_key_vault           = "s105d01-kv"                        # Currently Set to Development while waiting on Staging Service Now ticket to be actioned
 azure_resource_group      = "s105d01-dev-vault-resource-group"  # Currently Set to Development while waiting on Staging Service Now ticket to be actioned
+alerts = {
+  SchoolExperience_Staging = {
```

Comment:
Some of these are defaults and we can simplify be removing from the resource and the config. See an example:
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/terraform/workspace_variables/production.tfvars#L22
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/terraform/modules/statuscake/main.tf#L15

Reply:
Changed to look the same